### PR TITLE
[APPSRE-10253] - Add validation for rds enable blue_green_update

### PR DESCRIPTION
**Pull request by ninabauer**
Adds validation for rds enable blue_green_update

### CODE REVIEW

File: reconcile/utils/terraform_client.py [hunk 1]
```diff
@@ -785,6 +785,56 @@ def validate_db_upgrade(
...
+        account_name,
+        engine,
+        version,
+        parameter_group,
+        replica_source,
+        region_name,
+    ):
+        supported_versions = {
+            "mysql": ["5.7", "8.0.15"],
+            "postgres": ["11.21", "12.16", "13.12", "14.9", "15.4", "16.1"],
```
Comment (patjlm) [inline reconcile/utils/terraform_client.py:810]: Can we avoid hard coding these lists of versions somehow?

Comment (ninabauer) [reply reconcile/utils/terraform_client.py]: I tried to find an api that provided information specific to blue/green deployment compatibility kind of like how we used DescribeDBEngineVersions for a valid upgrade target but I did not find anything. Do you have any other ideas for how we could avoid hard coding the versions?

Comment (BumbleFeng) [reply reconcile/utils/terraform_client.py]: I think the only way here is hard coding.

**Review by BumbleFeng — APPROVED** [review]

**Pull request merged**